examples/model_serving/cv/table-transformer/README.md

```diff
@@ -0,0 +1,67 @@
+
+Table Transformer is actually a DETR model - by Microsoft Research (which is part of Transformers) to perform table detection and table structure recognition on documents.
+
+"DETR is short for DEtection TRansformer, and consists of a convolutional backbone (ResNet-50 or ResNet-101) followed by an encoder-decoder Transformer. It can be trained end-to-end to perform object detection (and panoptic segmentation). The main contribution of DETR is its simplicity: compared to other models like Faster R-CNN and Mask R-CNN, which rely on several highly engineered things like region proposals, non-maximum suppression procedure and anchor generation, DETR is a model that can simply be trained end-to-end, and fine-tuned just like you would fine-tune BERT. This is possible due to the use of a clever loss function, the so-called bipartite matching loss." 
+
+---- From [Jupyter notebook](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/Table%20Transformer/Using_Table_Transformer_for_table_detection_and_table_structure_recognition.ipynb)
+
+Steps to run the example:
+
+1. Create a new notebook server on **Kubeflow on vSphere** dashboard with 1 CPU and 4G RAM using custom Docker image `projects.registry.vmware.com/models/notebook/inference:cv-pytorch-cpu-v1`. GPU is not needed. 
+
+2. `CONNECT` to the notebook server and launch a Terminal in the created notebook server.
+
+3. Download the [Jupyter notebook](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/Table%20Transformer/Using_Table_Transformer_for_table_detection_and_table_structure_recognition.ipynb) (hash value is`c95e0a9` in case updated version doesn't work and you need to use this version) in the Terminal via command: 
+
+   ```shell
+   wget https://github.com/NielsRogge/Transformers-Tutorials/blob/d1872b1b60a9785c389d5c2b0261ffee55cc1eda/Table%20Transformer/Using_Table_Transformer_for_table_detection_and_table_structure_recognition.ipynb
+   ```
+
+4. Open the downloaded notebook file.
+
+5. Remove option `-q` from the following two cells in order to see the installation process and make sure the expected modules are installed:
+
+   - `!pip install -q git+https://github.com/huggingface/transformers.git`
+   - `!pip install -q timm`
+
+   You may use the following command in the Terminal to check if the expected Python module and version is installed:
+
+   ```sh
+   pip list | grep <module name>
+   ```
+
+6. Make sure the installed torch version is above `1.9.1`(not included) as the example doesn't work with lower versions. The following versions are tested that are working:
+
+   - 1.10.0
+
+   - 1.10.1
+
+   - 1.10.2
+
+   - 1.11.0
+
+   - 1.12.0
+
+   - 1.12.1
+
+   - 1.13.0
+
+   - 1.13.1
+
+   - 2.0.0
+
+   - 2.0.1
+
+   Run `!pip install torch==<version number>`  in a new cell  after `!pip install timm` in the notebook.
+
+7. Run the following commands to avoid problem of downloading (model and data)  from [Hugging Face](https://huggingface.co):
```

Comment:
Suggest to move this step to the bottom as a trouble shooting suggestion instead of a necessary step.  Most likely this is not an issue for users in other country.

Reply:
This is a common issue with Hugging Face, no matter where the users are.

Reply:
It's OK, just why the author didn't add this code. I believe the author has tested the code before publishing. That's why I do not suggest to use this as a necessary step.

Reply:
All the notebooks in the repository including the one we discuss are run in colab. There is no such issue in colab.

The issue is discussed at https://github.com/huggingface/transformers/issues/17611